ote/src/cljs/ote/views/route.cljs

```diff
@@ -64,12 +67,16 @@
                             (:name next)))
                         (partition-all 2 1 wizard-steps))]
     [:div.route
-     [ui/stepper {:active-step i}
+     [ui/stepper {:active-step i
+                  :connector (r/as-element [ic/navigation-arrow-forward])}
       (doall
-       (for [{label :label} wizard-steps]
-         ^{:key label}
-         [ui/step
-          [ui/step-label label]]))]
+        (for [{label :label current-step :name} wizard-steps]
+          ^{:key label}
+          [ui/step (when (rc/validate-all-steps route current-step wizard-steps)
```

Comment:
Calculate validity once and put it in the `for` :let, not it is repeated twice: once for style and once for click handler